packages/@react-aria/calendar/test/useCalendar.test.js

```diff
@@ -141,4 +141,41 @@ describe('useCalendar', () => {
       await testKeyboard(new CalendarDate(2019, 6, 5), 'June 2 to 15, 2019', 'End', 1, 'Saturday, June 8, 2019', 'June 2 to 15, 2019', {visibleDuration: {weeks: 2}});
     });
   });
+
+  async function testPagination(defaultValue, rangeBefore, rangeAfter, rel, count,  visibleDuration, pageDuration) {
+    let {getByRole, getByLabelText, unmount} = render(<Example defaultValue={defaultValue} autoFocus visibleDuration={visibleDuration} pageDuration={pageDuration} />);
+    let grid = getByRole('grid');
+    expect(grid).toHaveAttribute('aria-label', rangeBefore);
+
+    let btn = getByLabelText(rel);
+
+    for (let i = 0; i < count; i++) {
+      fireEvent.click(btn);
+    }
+
+    expect(grid).toHaveAttribute('aria-label', rangeAfter);
+
+      // clear any live announcers
+    act(() => {
+      jest.runAllTimers();
+    });
+  
+    unmount();
```

Comment:
please put these into an afterEach, i'll include how to structure the test in my next comment such that that works

Reply:
I just wanted the tests to be consistent with the existing ones. I guess, we should refactor the existing one, too.